src/20240723_AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnershipPhase2/config.ts

```diff
@@ -0,0 +1,15 @@
+import {ConfigFile} from '../../generator/types';
+export const config: ConfigFile = {
+  rootOptions: {
+    pools: ['AaveV3Ethereum'],
+    title: 'TokenLogic & karpatkey Service Provider Partnership-phase 2',
+    shortName: 'TokenLogicKarpatkeyServiceProviderPartnershipPhase2',
+    date: '20240723',
+    author: 'TokenLogic & karpatkey',
+    discussion:
+      'https://governance.aave.com/t/arfc-tokenlogic-karpatkey-financial-service-providers-phase-ii/18285',
+    snapshot: '',
```

Comment:
Could you add a link for a snapshot here?

Reply:
link added